.github/PULL_REQUEST_TEMPLATE/chore_template.md

```
@@ -0,0 +1,15 @@
+### PR 타입
+
+CHORE : 코드 포맷팅, 세미콜론 누락, 코드 변경이 없는 경우
+
+### 반영 브랜치
+
+ex) feat/login -> dev
+
+### 변경 사항
+
+#이슈번호 or JIRA ID
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)